.kokoro/build.sh

```diff
@@ -99,10 +99,13 @@ if [[ $SQL_CLIENT ]]; then
 	wget --quiet https://dl.google.com/cloudsql/cloud_sql_proxy.linux.amd64 -O cloud_sql_proxy
 	chmod +x cloud_sql_proxy	
 	if [[ $SQL_CLIENT == 'sqlserver' ]]; then
-		./cloud_sql_proxy -instances="${CONNECTION_NAME}"=tcp:1433 &>> cloud_sql_proxy.log &
+		./cloud_sql_proxy -instances="${INSTANCE_CONNECTION_NAME}"=tcp:1433 &>> cloud_sql_proxy.log &
+	elif [[ $SQL_CLIENT == 'mysql' ]]; then
+		export DB_HOST=127.0.0.1:3306
+		./cloud_sql_proxy -instances="${INSTANCE_CONNECTION_NAME}"=tcp:3306 &>> cloud_sql_proxy.log &
 	else
-		mkdir /cloudsql; chmod 777 /cloudsql 
```

Comment:
Did something change about needing to pre-create & permission this directory?

Reply:
I changed it to run the tests using TCP instead, so there's no need for a socket directory. I noticed that there were issues with writing to the root directory when the tests started failing once I changed them to run against an actual db connection

Reply:
Got it, so we're not doing end-to-end tests where the code is deployed to GAE or Cloud Run?

Reply:
Not yet, anyway. Setting up GAE and Cloud Run e2e tests for all Cloud SQL samples is on the roadmap.